src/senaite/sync/browser/views.py

```diff
@@ -65,13 +65,19 @@ def __call__(self):
             url = storage["credentials"]["url"]
             username = storage["credentials"]["username"]
             password = storage["credentials"]["password"]
-            content_types = storage["configuration"].get("content_types", None)
+            content_types = storage["configuration"].get("content_types", [])
+            unwanted_content_types = storage["configuration"].get("unwanted_content_types", [])
+            prefix = storage["configuration"].get("prefix", None)
+            prefixable_types = storage["configuration"].get("prefixable_types", [])
```

Comment:
Same function suggested above to be called from the tpl, could be used here to get all these settings in an easier way